scrapers/poland.py

```diff
@@ -143,8 +144,7 @@ def get_reception_points_pl(self, soup):
         item.pop(0)
         # TODO: Remove the entire above block if and when they fix the formatting on the site.
 
-        count = 0
-        for i in item:
+        for count, i in enumerate(item):
```

Comment:
Loop gets incremented a second time by the `count += 1` at the end

Reply:
will fix in next commit